lang/py/avro/io.py

```diff
@@ -419,7 +413,7 @@ class BinaryEncoder:
 
     def __init__(self, writer: IO[bytes]) -> None:
         """
-        writer is a Python object on which we can call write.
+        Writer is a Python object on which we can call write.
```

Comment:
Since this refers to the "writer" parameter, it should remain lowercase. But you could consider backticks, as in…

```
        """
        `writer` is an object on which we can call `write`.
        """
```